lib/component.js

```diff
@@ -172,6 +172,16 @@
     // Use `nextProps` or `component.props` and grab `model` and `collection`
     // from there
     var props = nextProps || component.props || {};
+    if (component.overrideModel && typeof component.overrideModel === 'function'){
```

Comment:
Your right. I'm going to rename the methods do `getOverridenModel` and `getOverridenCollection`. Looks good? I'll work also on a couple of unit tests. I'll cut a new beta release ASAP.